cdc/sink/codec/avro.go

```diff
@@ -300,7 +300,7 @@ func getAvroDataTypeFallback(v interface{}) (string, error) {
 var unsignedLongAvroType = avroLogicalType{
 	Type:        "bytes",
 	LogicalType: decimalType,
-	Precision:   8,
+	Precision:   64,
```

Comment:
Could you add some reference about this change?

Reply:
Precision `8` is not enough to hold the whole longlong type, this is discussed in #4420 . Confluent schema registry use `64` as the default precision. https://github.com/confluentinc/schema-registry/blob/ba073ebb2bd655e0c0e316a54d355a65c48246e3/avro-data/src/main/java/io/confluent/connect/avro/AvroData.java#L279

Reply:
Got it! Could you add the comments in code?